magefile.go

```diff
@@ -265,11 +269,36 @@ func Unit() error {
 	mg.Deps(GenerateGoCode)
 
 	if mg.Verbose() {
-		return sh.RunV(goExecutableName, "test", "-v",
+		err := sh.RunV(goExecutableName, "test", "-v",
```

Comment:
What about integration Go-tests? "-tags integration".

Reply:
Done, I've updated `UnitFull` as well. I've also refactored unit test running to reduce repetition